.gitignore

```diff
@@ -106,5 +106,7 @@ images/
 ### import data ###
 *.csv
 
-    © 2022 GitHub, Inc.
+### pre commit ###
+*.yml
```

Comment:
1. A co jeżeli będziesz chciała dodać jakieś yamle do gita? By default nie będą widoczne jeżeli zrobisz `git status`
2. Pre-commit config powinien być załączony do repozytorium, żeby wszyscy programiści mieli te same standardy.

Reply:
OK, nie wiedziełam, że to powinno się wrzucać do repozytorium. Myślałam że to moje wewnętrzne czyszczenie kodu. W takim razie usunę z gitignore i zpushuję na githuba